templates/settings.tpl

```diff
@@ -15,18 +15,21 @@
 	{rdelim});
 </script>
 
-{if $prerequisitesMissing|@count > 0}
-	<ul>
-		{foreach from=$prerequisitesMissing item=message}
-			<li><span class='pkp_form_error'>{$message}</span></li>
-		{/foreach}
-	</ul>
-{/if}
-
 <div id="plnSettings">
 	<form class="pkp_form" id="plnSettingsForm" method="post" action="{url router=$smarty.const.ROUTE_COMPONENT op="manage" category="generic" plugin=$pluginName verb="settings" save=true}">
 		{include file="controllers/notification/inPlaceNotification.tpl" notificationId="PLNSettingsFormNotification"}
 
+		{if $prerequisitesMissing|@count > 0}
```

Comment:
I see the button here, but the style is really off :)

![image](https://github.com/pkp/pln/assets/361921/472eb1cd-5961-4c87-abd1-eed46cb7e676)

Reply:
About the "user support", I wanted to display the log of the task, but as it's shared with all journals, it's not so interesting, it will need some improvements :)